packages/grafana-ui/src/components/uPlot/config/UPlotConfigBuilder.ts

```diff
@@ -17,6 +17,24 @@ import { AxisPlacement } from '../config';
 import { pluginLog } from '../utils';
 import { getThresholdsDrawHook, UPlotThresholdOptions } from './UPlotThresholds';
 
+const cursorDefaults: Cursor = {
+  // prevent client-side zoom from triggering at the end of a selection
+  drag: { setScale: false },
+  points: {
+    /*@ts-ignore*/
```

Comment:
any idea why these all need ts-ignore?  are the defs wrong?

Reply:
the defs are share between the options objects you provide (which can be partial), and the objects you read back, which will have everything defined / defaulted.